docs/src/changelog/index.md

```diff
@@ -6,6 +6,7 @@ Versions and changes-within-versions are listed in reverse-chronological order:
 ## 0.1.11: 2022/06 - present
 
 @{verRC("0.1.11")}
++ 2022/08/03: Added support for `{!rsh} isDataVariant`.
```

Comment:
This should be under rc6 (but actually, you will be beat and it will end up being in rc7)

Reply:
Also, just say "Added ..."